userdocs/profiles.dev/src/components/Schema.js

```diff
@@ -0,0 +1,254 @@
+/*
+Copyright 2019 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+Modifications:
+  Copyright 2021 Weaveworks
+*/
+
+import React, {useState, useEffect, useRef} from 'react';
+import styles from '../css/schema.module.css';
```

Comment:
this import is not being used, you don't need CSS imported here anyway, please remove it :)

Reply:
> this import is not being used
yeh i haven;t figured out how to yet 🙈 

how do we get the CSS we want then?

Reply:
you just add new ccs styles inside `custom.css` file and it should take precedence over default styles :)

Reply:
nice 👍